### PR TITLE
Add schema

### PR DESCRIPTION
**Pull request by AbelG101**
## Changes
- Initialized the repo with Rails.
- Added migration files.
- Added schema for:
    - Users
    - Comment
    - Like
    - Post
- Added linters.

### CODE REVIEW

File: db/migrate/20221122190541_create_comments.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.integer :author
+      t.integer :post
+      t.text :text
```
Comment (m05e5) [inline db/migrate/20221122190541_create_comments.rb:6]: - [ ] You really did a great job. Just like you did with author, you have to set post as a foreign key to comment too
i.e (`AddPostsRefToComments`)

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @AbelG101 , 

Good job so far. !
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders


## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to improve

- [ ] Please check the comments below 

_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20221122190541_create_comments.rb (hunk 1 above)
Comment (AbelG101) [reply db/migrate/20221122190541_create_comments.rb:6]: @m05e5 Thank you for your feedback, I forgot about that 🤭

**Review by ShadyShawkat — APPROVED** [review]
## Hi @AbelG101, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- All linters are passing ✔️ 
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**